airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/JdbcConnector.java

```diff
@@ -15,6 +13,9 @@
 
 public abstract class JdbcConnector extends BaseConnector {
 
+  public static final String POSTGRES_CONNECT_TIMEOUT_KEY = "connectTimeout";
+  public static final Duration POSTGRES_CONNECT_TIMEOUT_DEFAULT_DURATION = Duration.ofSeconds(10);
```

Comment:
We don't want the core module to depend on the postgres jdbc driver just for constants. It could be a compile-only dep if anyone really cares about this but I don't see the point tbh.